ci/docker/alpine-linux-3.15

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.15
+
+RUN apk update; apk add g++ cmake git bash py3-pip libexecinfo-dev ninja curl-dev zlib-dev zip
```

Comment:
Using `apk add --no-cache` seems to be a [good way](https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk/49119046#49119046) to install dependencies when building a container image, as it pulls the latest versions while at the same time saving nothing to a local cache.